docs/send-data/opentelemetry-collector/data-transformations.md

```diff
@@ -13,8 +13,18 @@ You can find more detailed information about the available OTTL functions and th
 
 ### Example 1: Hashing a password in log body
 
+Consider the following example log
+
+`"username=user password=test"`
```

Comment:
@rnishtala-sumo should this be inside of the code block?

Reply:
thanks! put this inside a code block

Reply:
@rnishtala-sumo Oh sorry, to clarify, I was asking if that line belonged in the code block (above the `processors` line), or if it's meant to stand alone like that.

Reply:
its meant to be a separate line

Reply:
thanks for confirming!